### PR TITLE
Influxctl v2.8.0

### PR DESCRIPTION
**Pull request by powersj**
Closes #

_Describe your proposed changes here._

- [x] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [x] Rebased/mergeable

### CODE REVIEW

**Comment by powersj** [comment]
@sanderson,

I have pushed this with the version and change log updates. I've put a special note in the clustered change log. Should that be in both change logs to ensure it is seen?

File: content/influxdb/cloud-dedicated/reference/release-notes/influxctl.md [hunk 1]
```diff
@@ -11,6 +11,26 @@ menu:
...
 ---
 
+## v2.8.0 {date="2024-04-11"}
+
+### Features
+
+- Introduce the ability to query with InfluxQL
+- Add insecure configuration option to TLS config
+- Allow users to query system tables
+- Utilize to database proxy service
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/reference/release-notes/influxctl.md:21]:
```suggestion
- Introduce the ability to query with InfluxQL.
- Add insecure configuration option to TLS configuration.
- Allow users to query system tables.
- Utilize the database proxy service.
```

File: content/influxdb/clustered/reference/release-notes/influxctl.md [hunk 2]
```diff
@@ -12,6 +12,32 @@ weight: 202
...
+required this change. A user will observe an unimplemented gRPC error when on
+an older version of clustered. If upgrading is not possible, then users can
+continue to use a version prior to 2.8.0 until they are able to update.
+
+### Features
+
+- Introduce the ability to query with InfluxQL
+- Add insecure configuration option to TLS config
+- Allow users to query system tables
+- Utilize to database proxy service
```
Comment (sanderson) [inline content/influxdb/clustered/reference/release-notes/influxctl.md:28]:
```suggestion
- Introduce the ability to query with InfluxQL.
- Add insecure configuration option to TLS configuration.
- Allow users to query system tables.
- Utilize the database proxy service.
```

File: content/influxdb/clustered/reference/release-notes/influxctl.md [hunk 3]
```diff
@@ -12,6 +12,32 @@ weight: 202
 canonical: /influxdb/cloud-dedicated/reference/release-notes/influxctl/
 ---
 
+## v2.8.0 {date="2024-04-11"}
+
+`influxctl` 2.8.0 requires clustered version 20240326-922145 or newer. A change
+was made to how database commands were handled internally in clustered that
+required this change. A user will observe an unimplemented gRPC error when on
+an older version of clustered. If upgrading is not possible, then users can
+continue to use a version prior to 2.8.0 until they are able to update.
```
Comment (sanderson) [inline content/influxdb/clustered/reference/release-notes/influxctl.md:21]:
```suggestion
`influxctl` 2.8.0 requires InfluxDB Clustered version 20240326-922145 or newer.
A change was made to how database commands were handled internally in InfluxDB
Clustered that required this change. If using an older version of InfluxDB
Clustered, you will observe an unimplemented gRPC error when using functionality
dependent on the updated command handling. If upgrading is not possible, you can
continue to use a version prior to 2.8.0 until you are able to update.
```

**Review by sanderson — APPROVED** [review]
Made some suggestions. With those, this is good to go.

**Pull request merged**